osquery/core/system.cpp

```diff
@@ -86,6 +87,13 @@ FLAG(string,
 
 FLAG(bool, utc, true, "Convert all UNIX times to UTC");
 
+const std::vector<std::string> placeholderHardwareUUIDList{
```

Comment:
Can you use the `k` prefix for global consts?

Reply:
sure, my bad.